src/utils/engine.ts

```diff
@@ -13,9 +13,11 @@ export function getEngine(): AiEngine {
   
   if (provider?.startsWith('ollama')) {
     const ollamaAi = new OllamaAi();
-    const model = provider.split('/')[1];
-    if (model) ollamaAi.setModel(model);
-    
+    const model = provider.replace('ollama/', '');
```

Comment:
@xtliu97 wont the change from `split` to `replace` affect anything else?

Reply:
I've reviewed the code and realized that using replace could introduce issues. I’ve corrected it by using substring to ensure everything works correctly. Thanks for catching that!